docs/conf/portland/2019/news/events-activities.rst

```diff
@@ -0,0 +1,64 @@
+:template: {{year}}/generic.html
+:banner: _static/{{year}}/assets/headers/venue.png
+
+.. post:: March 25, 2019
+   :tags: tickets, events, practical, shirts, portland-2019
+
+Events and Activities, T-Shirts, and Tickets Update
+===================================================
+
+We're just under two months out, and we hope everyone is getting excited!
+
+In this update, we're happy to share with you the different events and activities that will be available as a part of the conference, announce our pop-up t-shirt shop, and remind you to grab your tickets before they run out (hint: very soon!).
+
+Remember to check out our `Welcome Wagon <https://www.writethedocs.org/conf/portland/2019/welcome-wagon/>`_ guide and the `Visiting Portland <https://www.writethedocs.org/conf/portland/2019/visiting/>`_ page for help with planning your trip and making the most out of the conference.
```

Comment:
Welcome Wagon links to a page that says:
> The 2018 conference planning group includes an Explore Portland committee. Proud locals Mo Nishiyama and Alicia Duncan will be answering questions and providing tips and information on the #wtd-conferences Slack channel as the event draws near. Feel free to ping us directly as well!

(I could update the year, but not sure if any of the other info is also outdated?)

Reply:
Doesn't look this way from the website, I guess it was fixed already?

Reply:
Yes, it seems so :)